Palmer_McKay/detail.html

```diff
@@ -0,0 +1,33 @@
+<%inherit file="app_base.htm" />
+
+
+
+<%block name="content_top">
+    <h1>${ prod_name | h}</h1>
+</%block>
+
+<%block name="content">
+
+    <div class="col-sm-1">
+      <ul class="img-list">
+        %for image in product.image_urls():
+        <li>
+          <img class="thumnail-img" src="${ image }" alt="Thumnail Image for ${ product.name }" />
+        </li>
+        %endfor
+      </ul>
+    </div>
+    <div class="col-sm-7">
```

Comment:
col-md-7 might be better. I mean, unless you're using a normal spinning disk hard drive on a non hd laptop.

Reply:
I will take that into consideration.